### DISCUSSION TITLE
Comparison with "Apache Tika" via command line possible?

### DISCUSSION THREAD

**Discussion by martinhennecke69martinhennecke69** [WinMerge/winmerge#2424] [General]
I've seen that you can compare PDF files very well with "Apache Tika". Is there also a way to perform a PDF comparison with "Apache Tika" via command line?

**Comment by sdottaka** [comment]
If you add `/unpacker ApacheTika` to the command line arguments as shown below, the ApacheTika plugin will be used to compare files.

~~~
"c:\Program Files\WinMerge\WinMergeU.exe" c:\tmp\file1.pdf c:\tmp\file2.pdf /unpacker ApacheTika
~~~

**Comment by martinhennecke69martinhennecke69** [comment]
It worked, thank you very much :-)
